Clarify clock pins are down after scrambling (#65)
CarterKoala/wca-regulations-wrc: wca-guidelines.md

- 12a2+) [CLARIFICATION] For example, both Rw and 2Rw are valid notation for the same physical move for the 3x3x3 Cube. 1Rw is not valid notation for any NxNxN Cube, and 3Rw is not valid notation for the 2x2x2 Cube or the 3x3x3 Cube (but 3Rw is valid notation for the 4x4x4 Cube and larger NxNxN Cubes).
- 12g2+) [REMINDER] In the past, the penalty for changing state of the pins in inspection was a DNF. This is no longer the case.

> - 12g3+) [REMINDER] As clock scramble sequences end with such a move, all pins will always be down once the scramble sequence has been fully applied.

Reviewer: ```suggestion
- 12g3+) [REMINDER] As clock scramble sequences end with an X+ or X- move, all pins are pushed down once the scramble sequence has been fully applied.
```
